sketch.js

```diff
@@ -1,15 +1,81 @@
-const container = document.querySelector(".js-grid-container");
-addElement();
-const gridItem = document.querySelector(".js-grid-item");
-container.addEventListener("mouseover", function(e) {
-  const el = e.target;
-  el.classList.add("highlight");
-});
-
-function addElement() {
-  for (i = 0; i < 256; i++) {
-    const newDiv = document.createElement("div");
-    newDiv.classList.add("outline", "grid__item", "js-grid-item");
-    container.appendChild(newDiv);
-  }
-}
+(function() {
+  const container = document.querySelector(".js-grid-container");
+  const resizeButton = document.querySelector(".js-resize-grid");
+  const radioButton = document.querySelector(".js-radio-buttons");
+  let colorMode = 0;
+
+  addElement(16);
+
+  radioButton.addEventListener("change", function(e) {
+    colorMode = e.target.value;
+  });
+
+  container.addEventListener("mouseover", drawPixels);
+
+  resizeButton.addEventListener("click", function(e) {
+    const cellNumber = window.prompt("Enter a number 1-100?");
+    removeGrid();
+    addElement(cellNumber);
+  });
+
+  function addElement(n) {
+    for (i = 0; i < n; i++) {
+      const row = document.createElement("div");
+      row.classList.add("grid__row", "js-grid-row", "js-grid-item");
+      container.appendChild(row);
```

Comment:
DOM operations are much cheaper when done in memory (not attached to the page). I'd suggest to move `container.appendChild(row);` after the `for` loop. So that you build the entire row in memory and then attach everything at once.